tools/cli/commands/create.py

```diff
@@ -126,6 +126,13 @@
   mem_total_line=`cat /proc/meminfo | grep MemTotal`
   mem_total_value=`echo "${{mem_total_line}}" | cut -d ':' -f 2`
   memory_kb=`echo "${{mem_total_value}}" | cut -d 'k' -f 1 | tr -d '[:space:]'`
+
+  disk_kb_cutoff=`expr 10 "*" ${{memory_kb}}`
```

Comment:
If I understand your calculations correctly, this value is 10 not because it means 10%, but because 10 happens to go into 100% 10 times. I think it would be more clear to calculate the disk size * 10 / 100 and compare that against memory size. And maybe call the variable swap_size instead of memory_size, or set another variable swap_size to make it clear that you are creating a swap file that happens to be the same size as the amount of memory.

Reply:
No; the calculation is not trying to figure out if the swapfile size is < 10% of the disk size. Instead, it is trying to figure out if the disk size is > 10x the memory size.

The multiplier was chosen arbitrarily, but the check is about ensuring that the disk size is greater than some multiple of the memory size.

Reply:
I've added a comment inline in the code to try to make the logic easier to follow.

PTAL